### DISCUSSION TITLE
Using .ssX files

### DISCUSSION THREAD

**Discussion by parknich** [EmulatorJS/EmulatorJS#375] [Ideas]
Okay so the gba core uses mGBA however, even though this is the case, for whatever reason, .ssX files do not work. for example, i save a state in mgba desktop. it gives me "Pokemon Emerald Version.ss0" i try to load it in the website, and it doesnt work, it says its loaded but it actually doesnt do anything , so i try changing the file extension to .state but that doesnt work either.
Can you all add support for .ssx? Or atleast add support for using .sav files, as i would like to use my preexisting save.
Also this isn't an issue with my site as i also tried using the demo site, where it does the same thing ive described here. Thanks in advance

**Comment by Ramaerel** [comment]
I believe EmulatorJS has it's own save state format, which is why .ss* won't work. As for using .sav formats, there has been discussion on it before- I believe Ethan looked iinto doing it in emulatorJS and said it wasn't happening at the time, that might change with the new updates though? An option for saving/loading the .sav instead of .state. 

Anyway, for ways it has been done so far, check out these three issues: #235 #175 #52

**Comment by ethanaobrien** [comment]
Could you actually open an issue for this? I think this should be a feature. Sav files are currently saved into indexeddb
